Assets/MixedRealityToolkit/Interfaces/CameraSystem/IMixedRealityCameraSystem.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.CameraSystem
+{
+    public interface IMixedRealityCameraSystem : IMixedRealityEventSystem, IMixedRealityEventSource, IMixedRealityDataProvider { }
```

Comment:
All the public methods and properties should be defined here.

Reply:
At the moment there aren't any. Are you thinking we should cache the profile settings on initialize and then let devs modify those settings through the interface?

Reply:
Yes there is: `IsOpaque` It's defined in the profile, but shouldn't be.